Makefile

```diff
@@ -33,13 +33,15 @@ JOBS ?= $(shell nproc --all)
 BASE_OS ?= centos
 
 BASE_IMAGE ?= centos:7
+RELEASE_BASE_IMAGE ?= centos:7
 
 # do not change this; change versions per OS a few lines below (BASE_OS_TAG_*)!
 BASE_OS_TAG ?= latest
 
 BASE_OS_TAG_UBUNTU ?= 18.04
 BASE_OS_TAG_CENTOS ?= 7
 BASE_OS_TAG_CLEARLINUX ?= latest
+BASE_OS_TAG_REDHAT ?= 8.2
 
 INSTALL_RPMS_FROM_URL ?=
```

Comment:
Do we pass this value to Dockerfile.redhat anywhere? Is it possible to use make docker build and pass this parameter?